app/controllers/concerns/curation_concerns/curation_concern_controller.rb

```diff
@@ -93,26 +106,15 @@ def setup_form
       curation_concern.contributor << current_user.user_key
     end
 
-    # def verify_acceptance_of_user_agreement!
-    #   return true if contributor_agreement.is_being_accepted?
-    #   # Calling the new action to make sure we are doing our best to preserve
-    #   # the input values; Its a stretch but hopefully it'll work
-    #   self.new
-    #   respond_with(:curation_concern, curation_concern) do |wants|
-    #     wants.html {
-    #       flash.now[:error] = "You must accept the contributor agreement"
-    #       render 'new', status: :conflict
-    #     }
-    #   end
-    #   false
-    # end
-
     def _prefixes
       @_prefixes ||= super + ['curation_concerns/base']
     end
 
     def after_create_response
-      respond_with(main_app, :curation_concerns, curation_concern)
+      respond_to do |wants|
+        wants.html { redirect_to [main_app, :curation_concerns, curation_concern] }
+        wants.json { render json: curation_concern, status: :created }
```

Comment:
Rather than rendering the curation_concern itself, shoudln't this be rendering a jbuilder template? I.E.: 
https://github.com/rails/jbuilder/blob/master/lib/generators/rails/templates/show.json.jbuilder